retrybp/retry_test.go

```diff
@@ -94,23 +94,18 @@ func TestDoBatchError(t *testing.T) {
 		retry.Attempts(uint(len(errList))),
 	)
 
-	var batchErr errorsbp.Batch
-	if errors.As(err, &batchErr) {
-		if len(batchErr.GetErrors()) != len(errList) {
-			t.Errorf(
-				"wrong number of errors in %v, expected %v",
-				batchErr.GetErrors(),
-				errList,
-			)
-
-			for _, err := range errList {
-				if !errors.Is(batchErr, err) {
-					t.Errorf("%v is not wrapped by %v", err, batchErr)
-				}
+	if got, want := errorsbp.BatchSize(err), len(errList); got != want {
```

Comment:
is there something we can/should replace `errorsbp.BatchSize` with?

Reply:
see https://github.com/reddit/baseplate.go/commit/1edc07c27eff4cfcd00a5ff7bbc69e9d7549d076

basically it's much easier to use than handle a few different cases yourself (check nil, check not a batch error, then try to unwrap to a batch and get the size).